test/fast/FastTokenFacet.test.ts

```diff
@@ -107,6 +95,24 @@ describe('FastTokenFacet', () => {
       }
     });
 
+    // Reset expected returns.
+    spc.isMember.reset();
+    exchange.isMember.reset();
+    accessMock.isMember.reset();
+
+    // Add an extra member to the exchange.
+    exchange.isMember.whenCalledWith(exchangeMember.address).returns(true);
+    // Add a few Exchange and FAST members.
+    for (const { address } of [alice, bob, john]) {
+      exchange.isMember.whenCalledWith(address).returns(true);
+      accessMock.isMember.whenCalledWith(address).returns(true);
+    }
+    // Set the SPC member.
+    spc.isMember.whenCalledWith(spcMember.address).returns(true);
+    // Set the defaults when querying the SPC or Exchange fakes.
+    spc.isMember.returns(false);
+    exchange.isMember.returns(false);
```

Comment:
I moved these back out to here rightly/wrongly as they were running just once for the whole suite vs for each test case.

Reply:
Good catch!